src/sonic-config-engine/sonic-cfggen

```diff
@@ -298,7 +298,7 @@ def main():
     if args.var_json != None:
         if args.key != None:
             print(json.dumps(FormatConverter.to_serialized(data[args.var_json], args.key), indent=4, cls=minigraph_encoder))
-        else:
+        elif args.var_json in data:
```

Comment:
The `if` statement also references `data[args.var_json]`. Maybe this check should be done first, e.g.,

```
    if args.var_json != None and args.var_json in data:
        if args.key != None:
            print(json.dumps(FormatConverter.to_serialized(data[args.var_json], args.key), indent=4, cls=minigraph_encoder))
        else:
            print(json.dumps(FormatConverter.to_serialized(data[args.var_json]), indent=4, cls=minigraph_encoder))
```

Reply:
Done.